public/pages/Correlations/containers/CorrelationRules.tsx

```diff
@@ -80,8 +80,8 @@ export const CorrelationRules: React.FC<RouteComponentProps> = (props: RouteComp
 
   const onRuleNameClick = useCallback((rule: CorrelationRule) => {
     props.history.push({
-      pathname: ROUTES.CORRELATION_RULE_CREATE,
-      state: { rule, isReadOnly: true },
+      pathname: `${ROUTES.CORRELATION_RULE_EDIT}/${rule.id}`,
+      state: { rule, isReadOnly: false },
```

Comment:
Is this intended? Seems like it was supposed to create before

Reply:
Earlier we only supported view only using the create form and didn't update the URL to use the rule id. Now we are updating the URL to reflect that the action is edit.